platform/lego_nxt/cmds/snake/snake.c

```diff
@@ -138,7 +159,9 @@ static void refresh(void) {
 	for (i = 0; i < SIZE_Y; i++) {
 		for (j = 0; j < SIZE_X; j++) {
 			if (!space(field[f2d(j,i)])) {
-				point p = {j, i};
+				point p = {
+					j, i
```

Comment:
Is there control over this? Small structs are better looking in one line, while big ones should have every member initialization on separate line. I'd be fine if structs left untouched